frontend/test/metabase-visual/dashboard/fullscreen.cy.spec.js

```diff
@@ -0,0 +1,66 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATASET;
+
+const questionDetails = {
+  query: { "source-table": PRODUCTS_ID },
+};
+
+const filter = {
+  name: "Category",
+  slug: "category",
+  id: "ad1c877e",
+  type: "category",
+};
+
+describe("visual tests > dashboard > fullscreen", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createQuestionAndDashboard({ questionDetails }).then(
+      ({ body: { id, card_id, dashboard_id } }) => {
+        cy.addFilterToDashboard({ filter, dashboard_id });
+
+        // Connect filter to the card
+        cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
+          cards: [
+            {
+              id,
+              card_id,
+              row: 0,
+              col: 0,
+              sizeX: 12,
+              sizeY: 9,
+              visualization_settings: {},
+              parameter_mappings: [
+                {
+                  parameter_id: "ad1c877e",
+                  card_id,
+                  target: ["dimension", ["field-id", PRODUCTS.CATEGORY]],
+                },
+              ],
+            },
+          ],
+        });
+
+        cy.visit(`/dashboard/${dashboard_id}`);
+      },
+    );
+  });
+
+  it("renders in day mode and night mode", () => {
+    cy.icon("expand").click();
+
+    cy.icon("moon");
+
+    cy.percySnapshot("Shows dashboard in fullscreen day mode" + Date.now());
```

Comment:
Why did you add Date.now? I'm asking because Percy compares screenshots with the same names so if it is dynamic they will be considered as always newly added screenshots

Reply:
@alxnddr I had been playing with this in a personal Percy project, to learn how to leverage it before hitting CI, and for some reason it asked for a unique name.

Will remove.

Reply:
If you have a single test that has multiple calls of `percySnapshot` — the last one will take the name combined name of contexts + test name. In your case `visual tests > dashboard > fullscreen renders in day mode and night mode`
For multiple calls in the same test you can use
``` 
cy.percySnapshot('night');
...
cy.percySnapshot('day');
```

Reply:
@alxnddr 

Ah gotcha, makes sense. Updated.

Thank you for teaching me.